client/src/client.ts

```diff
@@ -18,6 +18,11 @@ export async function activate(context: ExtensionContext) {
                 workspace.createFileSystemWatcher('**/rebar.config'),
                 workspace.createFileSystemWatcher('**/rebar.lock')
             ]
+        },
+        initializationOptions: {
+           erlang: {
+               config_path: workspace.getConfiguration('erlang_ls').get('config.path'),
```

Comment:
If we use camelCase for `configPath` this could be `workspace.getConfiguration('erlang_ls').configPath,` instead.